src/components/Card.js

```diff
@@ -1,13 +1,34 @@
 export default class Card {
-  constructor({ name, link }, cardSelector, handleCardClick) {
+  constructor(
+    { name, link, owner, _id, likes },
+    cardSelector,
+    handleCardClick,
+    handleDeleteCardConfirmPopup,
+    handleLikeClick,
+    myID
+  ) {
     this.name = name;
     this.link = link;
+    this.ownerId = owner._id;
+    this.myID = myID;
+    this.imageID = _id;
+    this.likes = likes;
     this._cardSelector = cardSelector;
     this._handleCardClick = handleCardClick;
+    this._handleDeleteCardConfirmPopup = handleDeleteCardConfirmPopup;
```

Comment:
Please rename this variable to something like `_handleDeleteCard`, the card should not know anything about the confirmation popup